.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "FhirToDataLake/native/parquet/cpp/vcpkg"]
+	path = FhirToDataLake/native/parquet/cpp/vcpkg
```

Comment:
Do we need a `branch` option here to track the dedicated contents? Or it is meant to always track the latest `HEAD`?

Reply:
It will use the [commit](https://github.com/microsoft/FHIR-Analytics-Pipelines/pull/134/files#diff-f7552bb6842d6fb24280f9b371728329052fba18841231b57a23b7b312b36701) so the dependencies versions can be fixed.